tools/sourcemap-tools/src/ContentAppender.ts

```diff
@@ -0,0 +1,11 @@
+export class ContentAppender {
+    public appendToJSON(json: string, keyValues: object) {
```

Comment:
why we don't want to deserialize the json object and serialize it once again with new keys?

Reply:
wouldn't it be better if we do it once and keep the JSON file in memory for the source map generation?

Reply:
The source map is already generated at this point. I do not generate it - I only receive string concent.